tests/unit/type/zset.tcl

```diff
@@ -1652,6 +1652,21 @@ start_server {tags {"zset"}} {
         return $res
     }
 
+    # Check whether the zset members belong to the zset
+    proc check_member {mydict res} {
+        foreach ele $res {
+            assert {[dict exists $mydict $ele]}
+        }
+    }
+
+    # Check whether the zset members and score belong to the zset
+    proc check_member_and_score {mydict res} {
+       foreach {key val} $res {
+            assert {[dict exists $mydict $key]}
```

Comment:
i suppose this line is excessive (covered by the line below)

Reply:
right, has been deleted